### PR TITLE
[INJI-550] : refactor rendering issues

### PR DESCRIPTION
**Pull request by srikanth716**
Screenshot_2023-12-20-12-29-35-683_io mosip residentapp

### CODE REVIEW

File: components/ui/Header.tsx [hunk 1]
```diff
@@ -19,16 +19,29 @@ export const Header: React.FC<HeaderProps> = ({goBack, title, testID}) => {
...
-              name="arrow-left"
-              type="material-community"
-              onPress={goBack}
-              containerStyle={{
-                ...Theme.Styles.backArrowContainer,
-                marginLeft: 10,
-              }}
-              color={Theme.Colors.Icon}
-            />
+            {I18nManager.isRTL ? (
```
Comment (swatigoel) [inline components/ui/Header.tsx:22]: Did you check it's impact on other screen like QRConsent, MyBindedVCs?

File: components/ui/Modal.tsx [hunk 2]
```diff
@@ -30,14 +30,25 @@ export const Modal: React.FC<ModalProps> = props => {
...
             {props.arrowLeft ? (
-              <Icon
-                {...testIDProps('arrowLeft')}
-                name="arrow-left"
-                type="material-community"
-                onPress={props.onDismiss}
-                containerStyle={Theme.Styles.backArrowContainer}
-                color={Theme.Colors.Icon}
-              />
+              I18nManager.isRTL ? (
```
Comment (swatigoel) [inline components/ui/Modal.tsx:33]: Did you check it's impact on other screen like QRConsent, MyBindedVCs?

File: components/ui/Header.tsx (hunk 1 above)
Comment (srikanth716) [reply components/ui/Header.tsx:22]: verified on the both the QRConsent and MyBindedVCs screens working as eexpected

File: components/ui/Modal.tsx (hunk 2 above)
Comment (srikanth716) [reply components/ui/Modal.tsx:33]: verified on the both the QRConsent and MyBindedVCs screens working as expected

**Review by vijay151096 — APPROVED** [review]
lgtm

**Review by swatigoel — APPROVED** [review]

**Pull request merged**